native/SampleApps/RestAPIExplorer/RestAPIExplorer/AppDelegate.swift

```diff
@@ -69,21 +73,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // self.customizeLoginView()
         return true
     }
-
+    
```

Comment:
The indentation in this source was inconsistent enough I though a gentle re-indent would be OK.